lib/components-schema-v1_0_x.ts

```diff
@@ -117,6 +117,39 @@ export const componentsDefinitionSchema_v1_0_x = {
                                     },
                                 },
                             },
+                            {
+                                additionalProperties: false,
+                                required: ['type', 'options'],
+                                properties: {
+                                    type: {
+                                        enum: ['radio'],
+                                        description: 'Radio control type with fixed number of options',
+                                    },
+                                    options: {
+                                        type: 'array',
+                                        minItems: 1,
+                                        items: {
+                                            type: 'object',
+                                            additionalProperties: false,
+                                            required: ['caption', 'icon'],
+                                            properties: {
+                                                caption: {
+                                                    type: 'string',
+                                                    description: 'Label of the item',
+                                                },
+                                                icon: {
```

Comment:
Should we do something on the backend for the icon? Similar to the way component icons are converted to urls?

Reply:
Definitely